sdks/python/apache_beam/examples/wordcount_it_test.py

```diff
@@ -64,6 +65,18 @@ def test_wordcount_it(self):
     # and start pipeline job by calling pipeline main function.
     wordcount.run(test_pipeline.get_full_options_as_args(**extra_opts))
 
+  @attr('IT')
+  def test_wordcount_fnapi_it(self):
+    test_pipeline = TestPipeline(is_integration_test=True)
+
+    pipeline_verifiers = [PipelineStateMatcher()]
+    extra_opts = {'experiment': 'beam_fn_api',
+                  'on_success_matcher': all_of(*pipeline_verifiers)}
+
+    # Get pipeline options from command argument: --test-pipeline-options,
+    # and start pipeline job by calling pipeline main function.
+    wordcount_fnapi.run(test_pipeline.get_full_options_as_args(**extra_opts))
```

Comment:
Any reason not to just write

test_pipeline.get_full_options_as_args(
    experiment='beam_fn_api',
    on_success_matcher=PipelineStateMatcher())

Reply:
Thank you, fixed.